docs/development/writingzeppelininterpreter.md

```diff
@@ -63,12 +60,12 @@ Once you have build your interpreter, you can place your interpreter under direc
 
 To configure your interpreter you need to follow these steps:
 
-1. create conf/zeppelin-site.xml by copying conf/zeppelin-site.xml.template to conf/zeppelin-site.xml 
+1. Create conf/zeppelin-site.xml by copying conf/zeppelin-site.xml.template to `conf/zeppelin-site.xml `.
 
-2. Add your interpreter class name to the zeppelin.interpreters property in conf/zeppelin-site.xml
+2. Add your interpreter class name to the zeppelin.interpreters property in `conf/zeppelin-site.xml`.
 
-  Property value is comma separated [INTERPRETER_CLASS_NAME]
-for example,
+  Property value is comma separated [INTERPRETER\_CLASS\_NAME].
```

Comment:
nit: We can remove escaping character and write `[INTERPRETER_CLASS_NAME]`. But it is a kind of preference so it's up to you.